presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadata.java

```diff
@@ -322,4 +363,9 @@ public void dropSchema(ConnectorSession session, String schemaName)
     {
         jdbcClient.dropSchema(JdbcIdentity.from(session), schemaName);
     }
+
+    private static boolean containSameElements(Iterable<? extends ColumnHandle> first, Iterable<? extends ColumnHandle> second)
+    {
+        return ImmutableSet.copyOf(first).equals(ImmutableSet.copyOf(second));
```

Comment:
What about order here?

Reply:
I can change it, but it doesn't really matter here. There's not much benefit in accepting a pushdown if it only changes the order of the fields (other than eliminating an almost-free projection from the plan)

Reply:
Actually, we have to keep it like this for now. There's something going on in PushProjectionsIntoTableScan that's not preserving the order as it extracts the expressions to project. As a result, the plan never converges. I'll look into it separately and change this later when it's fixed.